pinot-spi/src/main/java/org/apache/pinot/spi/filesystem/PinotFS.java

```diff
@@ -143,7 +143,7 @@ long length(URI fileUri)
 
   /**
    * Copies a file from a remote filesystem to the local one. Keeps the original file.
-   * @param srcUri location of current file on remote filesystem
+   * @param srcUri location of current file on remote filesystem (must not be a directory)
```

Comment:
Do we have any usage that requires directory copy?
Shall we consider add a new API like `copySingleFile(...)` or add a flag parameter `isRecursive`?

Reply:
Yes, will update the PR